<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyHome</class>
 <widget class="QScrollArea" name="DailyHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel{
	font:23px;
}
/*
QFrame[frameShape=&quot;4&quot;][frameShadow=&quot;48&quot;]
{
    border: none;
	background: #dddddd;
	max-height: 1;
}
QFrame[frameShape=&quot;5&quot;][frameShadow=&quot;48&quot;]
{
    border: none;
    background: #dddddd;
	max-width: 1;
}

QAbstractButton {
background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:45px;
 border-color: lightgray;
 max-width:90px;
 max-height:90px;
 min-width:90px;
 min-height:90px;
}*/



QAbstractButton
{
    min-width: 330px;
	min-height: 30px;
}
</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>17</y>
     <width>920</width>
     <height>600</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>13</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <item>
       <widget class="QLabel" name="iconLabel1">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="itrac.qrc">:/res/wash-40.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>去污区</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout2">
      <item>
       <widget class="QToolButton" name="recycleButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>回收</string>
        </property>
        <property name="icon">
         <iconset resource="itrac.qrc">
          <normaloff>:/res/910001.png</normaloff>:/res/910001.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>220</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="washButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>清洗</string>
        </property>
        <property name="icon">
         <iconset resource="itrac.qrc">
          <normaloff>:/res/910006.png</normaloff>:/res/910006.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>220</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout3">
      <item>
       <widget class="QLabel" name="iconLabel2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="itrac.qrc">:/res/pack-40.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>检查包装及灭菌区</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout4">
      <item>
       <widget class="QToolButton" name="washCheckButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>配包</string>
        </property>
        <property name="icon">
         <iconset resource="itrac.qrc">
          <normaloff>:/res/910003.png</normaloff>:/res/910003.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>220</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="sterileButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>灭菌</string>
        </property>
        <property name="icon">
         <iconset resource="itrac.qrc">
          <normaloff>:/res/910004.png</normaloff>:/res/910004.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>220</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout5">
      <item>
       <widget class="QLabel" name="iconLabel3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="itrac.qrc">:/res/storage-40.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>无菌物品存放区</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout6">
      <item>
       <widget class="QToolButton" name="sterileCheckButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>灭菌审核</string>
        </property>
        <property name="icon">
         <iconset resource="itrac.qrc">
          <normaloff>:/res/910005.png</normaloff>:/res/910005.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>220</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="dispatchButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>发放</string>
        </property>
        <property name="icon">
         <iconset resource="itrac.qrc">
          <normaloff>:/res/910006.png</normaloff>:/res/910006.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>220</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="itrac.qrc"/>
 </resources>
 <connections/>
</ui>
